install.sh

```diff
@@ -95,8 +95,10 @@ function install() {
   echo "[PRE-INSTALL]: GATEWAY_IP is set to: $GATEWAY_IP"
 
 
+  # shellcheck disable=SC2034
```

Comment:
I would write a comment about why you are disabling a specific check, to avoid me - and you next time - having to go, read the documentation and wonder about why the exception was added.

Additionally, you could probably avoid this specific issue with:
```
set_variable_from_command POD_CIDR "${POD_CIDR_COMMAND}" "[ERROR]: Unable to find POD_CIDR"
```
and adapting the `set_variable_from_command` function accordingly